### PR TITLE
Mass conversion insert

### PR DESCRIPTION
**Pull request by hammerhead**

### CODE REVIEW

File: api_versions.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ admin:
     conversion_dispositions: 2
     decrypt_affiliate_link: 1
     mass_conversion_adjustment: 2
-    mass_conversion_insert: 2
+    mass_conversion_insert: 1
```
Comment (dziemba) [inline api_versions.yml]: Why don't we use v2?

Comment (hammerhead) [inline api_versions.yml]: Version 2 returns a "Unable to handle request without a valid action parameter. Please supply a valid soap action." error when using it's WSDL... Version 1 works fine for the same request.

Comment (dziemba) [inline api_versions.yml]: Okay. Can you add a comment that v2 is broken?

File: spec/lib/soapy_cake/admin_track_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+RSpec.describe SoapyCake::AdminTrack do
```
Comment (dziemba) [inline spec/lib/soapy_cake/admin_track_spec.rb]: Having unit and integration tests in the same spec file seems weird...

Comment (hammerhead) [inline spec/lib/soapy_cake/admin_track_spec.rb]: I cleaned up the whole specs and separated them everywhere.

Comment (dziemba) [inline spec/lib/soapy_cake/admin_track_spec.rb]: Don't see it?! You just renamed some files.

Comment (hammerhead) [inline spec/lib/soapy_cake/admin_track_spec.rb]: Look closer :eyes:

**Pull request merged**